src/sdk/pynni/nni/compression/torch/utils/counter.py

```diff
@@ -44,6 +46,11 @@ def count_flops_params(model: nn.Module, input_size, verbose=True):
     inputs = torch.randn(input_size).to(device)
 
     hook_module_list = []
+    if custom_ops is None:
+        custom_ops = {}
+        
+    custom_ops.update(custom_mask_ops)
```

Comment:
Should this be done in the opposite way: `custom_op` override `custom_mask_ops`?